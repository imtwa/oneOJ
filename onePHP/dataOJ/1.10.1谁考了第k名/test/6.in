92 38
6376611 92.1305
4985762 87.0826
1519393 37.1916
5228507 87.7129
4483287 30.7318
4272649 90.1781
3741395 66.8902
257483 84.0287
4488758 65.902
3029106 41.8171
3523804 22.3045
1031571 44.214
414232 92.935
5062628 90.8544
4404779 26.0798
2035717 90.7776
3380634 56.0815
1900901 48.8815
4482385 53.4191
6574098 64.4291
22913 32.4635
3038794 25.3073
321791 72.1808
3740461 45.4181
139680 77.3132
5528819 97.5177
3555738 69.4192
2431870 87.8851
5720794 78.342
6497669 32.523
6460336 61.3674
5366147 88.7227
5415703 81.1558
461710 27.9376
4116830 66.8105
2074225 51.2502
1950073 36.3163
5934019 79.8216
5426773 68.6445
4189547 42.6708
6004308 98.0482
6706132 53.1804
5488972 25.9624
4337739 85.7393
61665 83.1785
161067 85.8576
6061816 53.3083
4584549 35.0561
764194 41.017
2929976 34.6559
169203 73.5861
5346201 26.0216
451686 36.8018
5999862 49.1832
204026 37.4501
291133 80.2749
85761 33.8361
5590844 74.4437
6621261 76.8715
6890224 91.7564
853391 79.9262
5166500 92.8942
1132281 27.8529
5598594 82.3765
3672880 61.3291
5157727 92.8454
6381714 21.4767
1617254 65.8208
2591293 82.0573
3296716 85.7844
3323073 21.5092
394343 59.0157
4953571 49.8556
2039790 31.5813
551810 55.9612
6947087 91.3852
985055 45.8564
703290 42.7945
5890812 87.7665
6350959 83.6716
4221804 67.7262
4783151 90.0797
6124064 47.4159
5151368 26.2861
4401865 90.4839
5488840 87.713
4543320 20.8611
6410267 35.9693
6687276 24.5679
4603929 61.7963
6209014 20.5873
2764610 74.9465
