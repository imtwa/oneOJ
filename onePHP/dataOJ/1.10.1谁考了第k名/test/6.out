321791 72.1808